connect-inject/handler_ent_test.go

```diff
@@ -234,6 +299,17 @@ func TestHandler_MutateWithNamespaces(t *testing.T) {
 				}
 
 			}
+			// Check the patches, if provided.
+			if tt.ExpectedPatches != nil {
+				var actual []jsonpatch.JsonPatchOperation
+				if len(resp.Patch) > 0 {
+					require.NoError(json.Unmarshal(resp.Patch, &actual))
+					for i, _ := range actual {
+						actual[i].Value = nil
+					}
+				}
+				require.Equal(tt.ExpectedPatches, actual)
+			}
```

Comment:
This is great stuff, thanks! I'm with you on this change, I was hoping to piggyback because it had all the requisite setup pieces and fit okay, but it does get ugly you're right. I'll change this up!